rcljava/src/main/java/org/ros2/rcljava/node/BaseComposableNode.java

```diff
@@ -27,6 +27,11 @@ public BaseComposableNode(String name) {
     node = RCLJava.createNode(this.name);
   }
 
+  public BaseComposableNode(String name, String namespace) {
+    this.name = name;
```

Comment:
`this.name` can be deleted from the class, as it's not used.
I will ignore that in this review because it's unrelated to the PR.